requirements/requirements.in

```diff
@@ -1 +1,2 @@
 Django
+psycopg2==2.7.7
```

Comment:
please, add an empty row at the end of file

Reply:
@olyapetryshyn